README.md

```diff
@@ -1,20 +1,21 @@
 # Minehut Meta
-Welcome to the [Minehut](https://minehut.com) Meta GitHub repo! This is *the* place to **report bugs** and **suggest features and improvements** for Minehut, the server hosting network! You can also participate in conversations about feature requests and bug reports. 
+Welcome to the [Minehut](https://minehut.com) Meta GitHub repo! This is *the* place to **report bugs**, **suggest features and improvements**, and **request to modify the filter** for Minehut, the server hosting network! You can also participate in conversations about feature requests and bug reports. 
 
-We would like to improve communication with our users by providing official channels to report problems and tell us your ideas. This space will be taken as an (experimental) approach to see if you feel comfortable reporting and tracking your issues by means of this tool. This document (called the README) will serve as a starting point for making your first issue.
+We would like to improve communication with our users by providing official channels to report problems and tell us your ideas. This space will be taken as an approach to see if you feel comfortable reporting and tracking your issues by means of this tool. This document (called the README) will serve as a starting point for making your first issue.
```

Comment:
remove `(experimental)` as that seems way outdated with the over 900 issues total